### DISCUSSION TITLE
[macOS] Scrollbars aren't dark when certain system preferences are enabled

### DISCUSSION THREAD

**Discussion by januwland** [standardnotes/forum#2552] [General]
<img width="586" alt="Screen Shot 2021-01-07 at 1 54 07 pm" src="https://user-images.githubusercontent.com/38091842/103845259-d112ad00-50ef-11eb-866a-4f92fb59ddbb.png">

On any dark mode theme, MacOS scrollbars render in light mode. Would be excellent to have them in dark mode or native styling per theme.

**Comment by atmoio** [comment]
This seems to happen only if you have System Preferences > General > Show scroll bars > Always

But should be fixed nonetheless.

**Comment by JaspalSuri** [comment]
> This seems to happen only if you have System Preferences > General > Show scroll bars > Always

It also happens if it's set to `Automatically based on mouse or trackpad` (on macOS Big Sur 11.1) as well, but the scrollbars are dark when it's set to `When scrolling`. 

![image](https://user-images.githubusercontent.com/44656000/103909079-1915dd80-50b8-11eb-8619-708ac7a2c10e.png)
